judge/management/commands/runmoss.py

```diff
@@ -0,0 +1,46 @@
+from django.core.management.base import BaseCommand
+
+from judge.models import *
+
+from moss import *
```

Comment:
Third party library that are `pip install`ed should go before DMOJ modules.

Reply:
Now, no new line between `django` and `moss` since they are both third-party libraries.

Reply:
Also, you are only using `MOSS`, right? Why not just import `MOSS`?

Reply:
There are a bunch of `MOSS_LANG_*` constants defined.